apps/astarte_appengine_api/test/support/generators/group.ex

```diff
@@ -0,0 +1,47 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2017-2024 SECO Mind Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule Astarte.AppEngine.API.GroupTestGenerator do
+  @moduledoc """
+  Helpers group's generators
+  """
+  use ExUnitProperties
+
+  @max_subpath_count 10
+  @max_subpath_length 20
+
+  @doc """
+  Generate a random group name
+  Es.
+  world/europe/italy
+  """
+  def group_name do
```

Comment:
This generates a specific type of group name, not a generic one, so I'd rename it accordingly (e.g. `path_like_group_name` or `group_name_with_slash`)

Reply:
Not exactly. Generates _any_ type of group_name, with/without slash

Reply:
Right, I didn't consider the case where the list has 1 element, then it's fine